package.json

```diff
@@ -26,6 +27,8 @@
     "url": "https://github.com/versatica/JsSIP/issues"
   },
   "dependencies": {
+    "@types/debug": "^4.1.5",
+    "@types/node": "^13.13.5",
```

Comment:
Please use `@types/node" 14.x.x, the latest stable version.

Reply:
done